src/ranged.cpp

```diff
@@ -192,8 +190,16 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
         trajectory = g->m.find_clear_path( source, target );
     }
 
-    add_msg( m_debug, "%s proj_atk: shot_dispersion: %.2f",
-             disp_name().c_str(), shot_dispersion );
+    if( proj_effects.count( "MUZZLE_SMOKE" ) ) {
```

Comment:
Why do you duplicate [this](https://github.com/CleverRaven/Cataclysm-DDA/pull/17656/files#diff-c5873ed460ae46963dac594829494e52R214)?